### DISCUSSION TITLE
Purpose of <leader>da `+adapters`

### DISCUSSION THREAD

**Discussion by Jomik** [LazyVim/LazyVim#864] [Q&A] [answered]
Hey!
Just stumbled upon `<leader>da`, what do we use that for? I see nothing hooking into it?
Should we get rid of it? Or should we perhaps even make an equivalent for `extras.test`?

Currently for me it just results in a normal `da`, ignoring `<leader>` :)

**Comment by jyuan0** [comment, marked as answer]
Think it's for launching and connecting to the Lua debugger: https://www.lazyvim.org/plugins/extras/dap.nlua

I've been using it for language-specific debug stuff.

**Reply by Jomik** [reply]
Ahh of course 👍

**Reply by folke** [reply]
Feel free to file a bug for which-key, or send in a PR :)

Basically, groups without children should not be shown in which-key
